atlp-rwanda/atlp-devpulse-fn
# Fix ticket loading

### PR Description

This PR fixes loading and disabling buttons on the ticket pages and modals

### Description of tasks that were expected to be completed

The submission buttons should show a loader when submitting and be disabled for a better user experience

### How can this been tested?
- Clone repo
- Checkout branch `fix-ticket-loading`
- Make sure that you set the environment variables referring to `.env.example`
- Run `npm install`, then `npm run dev` to check if the app is running successfully
- Navigate to tickets for both an admin or applicant and you will see the loading when you do any submission

### Track PR (issue number & link)
N/A
